test/e2e/specs/test.spec.js

```diff
@@ -57,6 +57,15 @@ describe('Wallet App Test Cases', () => {
       });
     });
 
+    it('should view the bid from CLI', () => {
+      cy.exec('bash ./test/e2e/test-scripts/view-bids.sh', {
```

Comment:
`view-bids.sh` is essentially a 1 liner. How about just inlining it into the .js code?

Reply:
Also, I was running the test for `view.bids.sh`. And it tends to give the same issue solved here:
https://github.com/Agoric/agoric-sdk/pull/9024

But if I run the `agops inter bid list` from the shell, all works as expected. I reckon we're some using old dependencies in the `wallet-app`?

Reply:
> But if I run the agops inter bid list from the shell, all works as expected. I reckon we're some using old dependencies in the `wallet-app`?

I'm still trying to figure out why we're testing the bidding CLI code from agoric-sdk in the wallet-app repo at all.

But yes, we seem to be getting agoric-sdk (including the bidding CLI) from a published docker image for agoric-sdk... I suppose that's from a recent release.

https://github.com/Agoric/wallet-app/blob/8188ec6fb66c61bb43db9d1f57089487df70050b/Dockerfile#L1C1-L1C38

[agoric-sdk:latest](https://github.com/agoric/agoric-sdk/pkgs/container/agoric-sdk/206234664?tag=latest) ... looks like it's currently 20240420012145-[734e86](https://github.com/Agoric/agoric-sdk/commit/734e86) aka [agoric-upgrade-15-rc0](https://github.com/Agoric/agoric-sdk/releases/tag/agoric-upgrade-15-rc0) Apr 20.

The fix, 9024 wasn't merged until Apr 22.

The [instructions for bidding](https://docs.inter.trade/user-how-to/bidding-for-liquidated-collateral) don't say to use code from such a release. They say to do:

```
git clone https://github.com/Agoric/agoric-sdk
cd agoric-sdk
SKIP_DOWNLOAD=false ./bin/agd build
export PATH=$PWD/bin:$PATH
alias inter="yarn run --silent agops inter"
export AGORIC_NET=main
```

Installing straight from master is a little goofy (long story... agops is a test tool but we're using it as an end-user tool... cc @otoole-brendan ), but until we decide to do otherwise, I suppose we should do that in ci.

Reply:
The bids placed from the CLI have an element of UI testing as well. Once a bid is placed from the CLI, it is visible on the wallet-app UI. 
![image](https://github.com/Agoric/wallet-app/assets/60459172/01732d66-a067-45c0-8d2d-f6a20256fff3)

Reply:
Initially, I was installing directly from the master and it was goofy :D 
It also involved setting node and Golang in CI. The CI test runs took more than 15 minutes till I found `agoric-sdk` image.

Reply:
> I prefer to review as little `.sh` code as possible.
I'm working on eliminating bash scripts from liquidation tests, in [this pull request](https://github.com/Agoric/dapp-inter/pull/268). I'm also planning to apply a similar approach to the bid-related tests

Reply:
> Initially, I was installing directly from the master and it was goofy :D It also involved setting node and Golang in CI. The CI test runs took more than 15 minutes till I found `agoric-sdk` image.

Really? ci in the docs repo installs agoric-sdk from source and it runs in under 4min:
https://github.com/Agoric/documentation/actions/runs/8707389744/job/23882252835

I'm not sure what to tell you... if we want to test the product that we tell users to use for bidding, that's not the agoric-sdk image.